src/components/select/multi-select/multi-select-test.stories.tsx

```diff
@@ -646,3 +647,47 @@ export const OptionsWithSameName = () => {
   );
 };
 OptionsWithSameName.storyName = "Options with same name";
+
+export const AriaDescribedByExample = () => {
+  const MAX_SELECTIONS_ALLOWED = 2;
+  const [selectedPills, setSelectedPills] = useState([] as string[]);
+  const handleActivityChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    if (event.target.value.length <= MAX_SELECTIONS_ALLOWED) {
+      setSelectedPills(event.target.value as unknown as string[]);
+      partialAction("onChange")();
+    }
+  };
+  return (
+    <>
+      <MultiSelect
+        aria-describedby="comb-box-description"
```

Comment:
The description wasn't being read out but its just due to this small typo:
```suggestion
        aria-describedby="combo-box-description"
```

Reply:
Well spotted. I'll get it fixed.